### ISSUE TITLE
error: cannot bind to local control interface port

### ISSUE THREAD

**Issue by chanchan-wu** [zerotier/ZeroTierOne#1593]
when i use `zerotier-one -d` to make zeotier open, the error occur `zerotier-one: fatal error: cannot bind to local control interface port`.
however, `systemctl status zerotier-one.service`shows that it's activated:
`#systemctl status zerotier-one.service
zerotier-one.service - ZeroTier One
   Loaded: loaded (/lib/systemd/system/zerotier-one.service; enabled; vendor preset: enabled)
   Active: active (running) since Wed 2022-03-16 11:07:06 CST; 30s ago
 Main PID: 14603 (zerotier-one)
    Tasks: 3 (limit: 4915)
   CGroup: /system.slice/zerotier-one.service
           └─14603 /usr/sbin/zerotier-one

3月 16 11:07:06 a systemd[1]: Started ZeroTier One.`
also, the zerotier-cli function seems work fine:
`#zerotier-cli info
200 info 3b557e997c 1.4.6 ONLINE`
I try to kill zerotier and reopen it, but this error still there.
This prevents me from connecting to the server using ssh, showing `connect to host 10.147.17.xx port 22: No route to host`

A puzzling thing is that another device under the same network can connect perfectly. And my device can be connected when the network cable is disconnected and using the wifi under the same router, although it is very slow and stuck.

my device infomation here:
`uname -a
Linux a 4.15.0-171-generic #180-Ubuntu SMP Wed Mar 2 17:25:05 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
`
I have been using zerotier for a while, it allways works will, I hope you can help me solve this problem.

**Comment by glimberg** [comment]
that error means ZeroTier is already running, and your output from `systemctl status` shows that.  If you want to run zerotier from the command line with `zerotier-one -d`, then you'll need to stop the systemd managed service first with `systemctl stop zerotier-one`.

**Comment by chanchan-wu** [comment]
OHHH! I really appreciate your answer @[glimberg](https://github.com/glimberg). I stopped Zerotier with `systemctl stop zerotier-one` and restarted it as you said, and it succeeded. I had just been trying to kill the process. Thanks again. That helped me a lot.

**Comment by glimberg** [comment]
Yes.  By default zerotier is always running in the background after its installed.  you can use `systemctl (start|stop|restart) zerotier-one` to control it.  Running it manually with `zerotier-one -d` is not necessary.

**Comment by chanchan-wu** [comment]
Yes, I will remember it, thank you!

**Comment by chanchan-wu** [comment]
Hi @glimberg , unfortunately, my zerotier just works fine that one time(The speed was not ideal on that occasion).  A few hours passed and it became unusable again. Although I did use `systemctl (start|stop|restart) zerotier-one`, and now when I use this code, zerotier is not activated sometimes:
```
#systemctl stop zerotier-one
#systemctl start zerotier-one
#systemctl status zerotier-one.service
 zerotier-one.service - ZeroTier One
   Loaded: loaded (/lib/systemd/system/zerotier-one.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Thu 2022-03-17 20:14:02 CST; 98ms ago
  Process: 9077 ExecStart=/usr/sbin/zerotier-one (code=exited, status=1/FAILURE)
 Main PID: 9077 (code=exited, status=1/FAILURE)

3月 17 20:14:02 a systemd[1]: zerotier-one.service: Service hold-off time over, scheduling restart.
3月 17 20:14:02 a systemd[1]: zerotier-one.service: Scheduled restart job, restart counter is at 5.
3月 17 20:14:02 a systemd[1]: Stopped ZeroTier One.
3月 17 20:14:02 a systemd[1]: zerotier-one.service: Start request repeated too quickly.
3月 17 20:14:02 a systemd[1]: zerotier-one.service: Failed with result 'exit-code'.
3月 17 20:14:02 a systemd[1]: Failed to start ZeroTier One.
```
In this case, neither `stop` nor `restart` can make it active; it will only become `active` if I kill it and start it:
```killall -9 zerotier-one
# systemctl start zerotier-one
# systemctl status zerotier-one.service
● zerotier-one.service - ZeroTier One
   Loaded: loaded (/lib/systemd/system/zerotier-one.service; enabled; vendor preset: enabled)
   Active: active (running) since Thu 2022-03-17 20:21:11 CST; 1s ago
 Main PID: 9321 (zerotier-one)
    Tasks: 5 (limit: 4915)
   CGroup: /system.slice/zerotier-one.service
           └─9321 /usr/sbin/zerotier-one

3月 17 20:21:11 a systemd[1]: Started ZeroTier One.
```
But this situation is still can't connected. By the way, the `zerotier-one -d`still occur the error which the title noticed.  In the few hours I didn't change any setting. Any suggestion?
